### PR TITLE
[ADP-3476] Bump node dependency to version 10.1.2

### PR DESCRIPTION
**Pull request by paolino**
ADP-3476

### CODE REVIEW

File: justfile [hunk 1]
```diff
@@ -140,12 +140,12 @@ integration-tests match:
     -c integration-exe -j 2 --match="{{match}}"
 
 
-# start a shell with 10.1.1 in scope
+# start a shell with 10.1.2 in scope
 node-910:
```
Comment (Anviking) [inline justfile]: this is getting out of date, maybe we don't need it to contain the version number any longer as we don't test with two different node versions any more

**Review by Anviking — APPROVED** [review]

**Comment by HeinrichApfelmus** [comment]
🤔 Uhhhh, this

https://github.com/cardano-foundation/cardano-wallet/blob/f26e8d46e7c279db3a5d270b0917baa77d71d128/flake.nix#L145

should be

```
cardano-node-runtime.url = "github:IntersectMBO/cardano-node?ref=10.1.2"; 
```

, no? This is the version of the node that we are running integration tests against.

File: justfile (hunk 1 above)
Comment (Anviking) [reply justfile]: > cardano-node-runtime.url = "github:IntersectMBO/cardano-node?ref=10.1.2"; 
> , no? This is the version of the node that we are running integration tests against.

Oh, yes, this `node-910` actually appears to be unused 😅

**Comment by paolino** [comment]
Seems like I missed the bump to 10.1.1, bad. Thank you for looking into that

**Comment by michalrus** [comment]
@paolino @HeinrichApfelmus `flake.lock` is incorrect, you somehow stayed on `5d3da8ac771ee5ed424d6c78473c11deabb7a1f3`, which is still 9.2.1.

Reported as #4834.

**Pull request merged**